Sources/Sentry/SentryNetworkSwizzling.m

```diff
@@ -34,12 +35,21 @@ + (void)swizzleURLSessionTaskResume
 
 + (void)swizzleNSURLSessionConfiguration
 {
-    SEL httpAdditionalHeadersSelector = NSSelectorFromString(@"HTTPAdditionalHeaders");
-    SentrySwizzleInstanceMethod(NSURLSessionConfiguration.class, httpAdditionalHeadersSelector,
-        SentrySWReturnType(NSDictionary *), SentrySWArguments(), SentrySWReplacement({
-            return [SentryNetworkTracker.sharedInstance addTraceHeader:SentrySWCallOriginal()];
-        }),
-        SentrySwizzleModeOncePerClassAndSuperclasses, (void *)httpAdditionalHeadersSelector);
+    // iOS 13 doesn't have a method for HTTPAdditionalHeaders. Instead, it only has a property.
+    // Therefore, we need to make sure that NSURLSessionConfiguration has this method to be able to
```

Comment:
M: cant we swizzle a property?

Reply:
Not right now at the moment. We need to add the functionality or think of a different approach.